realm/realm-library/src/androidTest/java/io/realm/RealmConfigurationTests.java

```diff
@@ -796,6 +801,68 @@ public void rxFactory() {
         assertFalse(configuration2.getRxFactory() == dummyFactory);
     }
 
+    @Test
+    public void coroutinesFactory_defaultNotNull() {
```

Comment:
Missing test for what happens if you try to assign `null` to the factory. I assume we want to throw an IllegalArgumentException?